### PR TITLE
Pulling through Lesson 2.2 - Grid Printer

### PR DESCRIPTION
**Pull request by IanMayther**
Completed Lesson 2.2 after several commits.  Please review and provided feedback.

### CODE REVIEW

File: students/IanMayther/Lesson02/Exercise2.2.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        elif z >= x and z%x == 0:
+            line = line + " +"
+        else:
+            line = line + " -"
+    line = line + " +"
+    print(line)
+
+
+print_grid1(7)
+print_grid2(10, 5)
```
Comment (charlesrobison) [inline students/IanMayther/Lesson02/Exercise2.2.py:107]: Great!

File: students/IanMayther/Lesson02/Exercise2.3.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+for x in range(1, 101):
+    if x%3 == 0 and x%5 != 0:
+        print("Fizz")
+    elif x%3 != 0 and x%5 == 0:
+        print("Buzz")
+    elif x%3 == 0 and x%5 == 0:
+        print("FizzBuzz")
+    else:
+        print(x)
```
Comment (charlesrobison) [inline students/IanMayther/Lesson02/Exercise2.3.py:11]: Looks good

**Comment by IanMayther** [comment]
Good Morning Charles,

Thank you for the feedback.

Question regarding the pull request as it relates to submitting the
homework.  Can you only have one open pull request for a branch at a time?
That is to say from a homework submission perspective, I could use the same
pull request link for all of my activities submitted for a week?

I was struggling to have a second pull request after I created one of
Lesson 2.2.

I will get the rest of them submitted today.

Thanks,
Ian

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=icon>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=link>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by charlesrobison** [comment]
Hi Ian,

Yes, that's correct - you can only have one open PR at a time.  You can continue to add to the open PR if we haven't merged it and you're ready to move on, no problem.  Let me know if you need anything else.


Charles

**Pull request merged**